### ISSUE TITLE
Inadvertent sssd related calls in Leap 15.4

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2429]
During experimental preparation for our re-basing on openSUSE Leap 15.4 via pending pr #2427 for issue #2424, it was discovered that we appear, in 15.4 only, to be calling on sssd inadvertently, prior to its configuration for AD/LDAP function, resulting in massive slow-downs in group and some deeper user page views:
Reproducer - visit/refresh groups page:
```
Nov 15 16:06:23 rleap15-4 [RPM][21426]: install sssd-dbus-2.5.2-150400.4.11.1.x86_64: success
Nov 15 16:06:23 rleap15-4 [RPM][21426]: install sssd-dbus-2.5.2-150400.4.11.1.x86_64: success
Nov 15 16:06:23 rleap15-4 [RPM][21426]: Transaction ID 6373b8fe finished: 0
Nov 15 16:06:45 rleap15-4 dbus-daemon[656]: [system] Activating via systemd: service name='org.freedesktop.sssd.infopipe' unit='sssd-ifp.service' requested by ':1.33' (uid=0 pid=13385 comm="/usr/bin/python2 /opt/rockstor-dev/bin/gunicorn --")
Nov 15 16:06:45 rleap15-4 systemd[1]: Starting System Security Services Daemon...
Nov 15 16:06:45 rleap15-4 sssd[22235]: SSSD couldn't load the configuration database [2]: No such file or directory.
Nov 15 16:06:45 rleap15-4 systemd[1]: sssd.service: Main process exited, code=exited, status=4/NOPERMISSION
Nov 15 16:06:45 rleap15-4 systemd[1]: sssd.service: Failed with result 'exit-code'.
Nov 15 16:06:45 rleap15-4 systemd[1]: Failed to start System Security Services Daemon.
Nov 15 16:06:45 rleap15-4 systemd[1]: Dependency failed for SSSD IFP Service responder.
Nov 15 16:06:45 rleap15-4 systemd[1]: sssd-ifp.service: Job sssd-ifp.service/start failed with result 'dependency'.
Nov 15 16:06:45 rleap15-4 systemd[1]: Reached target User and Group Name Lookups.
Nov 15 16:06:45 rleap15-4 systemd[1]: sssd.service: Scheduled restart job, restart counter is at 1.
Nov 15 16:06:45 rleap15-4 systemd[1]: Stopped System Security Services Daemon.
Nov 15 16:06:45 rleap15-4 systemd[1]: Starting System Security Services Daemon...
Nov 15 16:06:45 rleap15-4 sssd[22240]: SSSD couldn't load the configuration database [2]: No such file or directory.
Nov 15 16:06:45 rleap15-4 systemd[1]: sssd.service: Main process exited, code=exited, status=4/NOPERMISSION
```
In the above we see that, as part of the initial diagnosis, we ended up installing the last of our sssd related dependencies before this issue was triggerable, the so-called InfoPipe, from:

> zypper info sssd-dbus

> Description    : 
>     Provides the D-Bus responder of sssd, called InfoPipe, which allows
>     information from sssd to be transmitted over the system bus.

**unit='sssd-ifp.service' requested by ':1.33' (uid=0 pid=13385 comm="/usr/bin/python2 /opt/rockstor-dev/bin/gunicorn --")**

which in turn finds our sssd not working, because there is no AD setup. Hence a suspected failing check on if AD / sssd is enabled.

Thanks to @FroggyFlox for assistance in the initial diagnosis of what might be going awry here.

**Comment by phillxnet** [comment]
Interestingly when we remove the pivital app in this case:
```
rleap15-4:~ # zypper remove sssd-dbus 
...
The following package is going to be REMOVED:
  sssd-dbus
...
rm: cannot remove '/var/lib/systemd/migrated/sssd-ifp': No such file or directory
```

And then re-run our cli reproducer for our observed delay in groups page visit:
```
rleap15-4:~ # dbus-send --print-reply --system --dest=org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe org.freedesktop.sssd.infopipe.GetUserGroups
Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.sssd.infopipe was not provided by any .service files
```
Re-indicating such a call does in fact now invoke our .service in a way it did not in Leap 15.3.

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2430
